dist/_locales/pl/messages.json

```diff
@@ -0,0 +1,202 @@
+{
+  "app_name": {
+    "message": "TabTabTab - Menedżer kart",
+    "description": "Nazwa aplikacji. Wyświetlana w web store."
+  },
+  "app_name_short": {
+    "message": "TabTabTab",
+    "description": "Skrócona nazwa aplikacji. Wyświetlana w górnej części menu rozwijanego."
+  },
+  "app_desc": {
+    "message": "To rozszerzenie usprawnia przeglądanie stron poprzez lepsze zarządzanie kartami.",
+    "description": "Krótki opis aplikacji. Wyświetlany w web store."
+  },
+  "window": {
+    "message": "Okno",
+    "description": ""
+  },
+  "current_window": {
+    "message": "Bieżące Okno",
+    "description": ""
+  },
+  "pinned": {
+    "message": "Przypięty",
+    "description": ""
+  },
+  "month": {
+    "message": "mies.",
+    "description": ""
+  },
+  "months": {
+    "message": "mies.",
+    "description": ""
+  },
+  "day": {
+    "message": "dzień",
+    "description": ""
+  },
+  "days": {
+    "message": "dni",
+    "description": ""
+  },
+  "hour": {
+    "message": "h",
+    "description": ""
+  },
+  "hours": {
+    "message": "h",
+    "description": ""
+  },
```

Comment:
Which one is more commonly used in Polish, 'h' or 'godziny'?
For example, in English, it is used in UI texts like '15 hours ago' to indicate elapsed time.

```json
  "hour": {
    "message": "godzina",
    "description": ""
  },
  "hours": {
    "message": "godziny",
    "description": ""
  },
```

Reply:
Yes, "godziny" is more commonly used, but it's **a little bit** more complicated too (I assume those messages try to give an answer to _when?_):

- _1 godzinę temu_
- _2 godziny temu_
- _8 godzin temu_
- _24 godziny temu_
- _36 godzin temu_
- ...

I mean - there are rules for that, but I wanted to keep it simple.
_Now_, I think **the best** solution is to use **_godz._** for both singular and plural.

Reply:
Thank you for letting me know!
Now let's go with `godz`!